docs/pathways/ontology-engineer.md

```diff
@@ -45,17 +44,11 @@ This section is a non-ordered collection of how to documents that an engineer mi
 
 - [Adding taxon restrictions](../howto/add-taxon-restrictions.md)
 - [Changing files in pull requests](../howto/change-files-pull-request.md)
-- [Cloning a repo](../howto/clone-mondo-repo.md)
 - [Creating new terms](../howto/create-new-term.md)
-- [Daily Curator Workflow](../howto/daily-curator-workflow.md)
+- [Daily Git Workflow (Curators and Engineers)](../howto/daily-curator-workflow.md)
```

Comment:
Changed this because even though the page is curator focused, it is a useful workflow to know for engineers as well. Added word 'Git' because it seems focused on largely on that.